web/src/components/CircleIcon.tsx

```diff
@@ -0,0 +1,58 @@
+import interview from "~public/icons/interview.svg";
+import check from "~public/icons/check.svg";
+import list from "~public/icons/list.svg";
+import {useState} from "react";
+import Image from 'next/image';
+
+interface CircleIconProps {
+  icon: string;
+  message: string;
+}
+
+export const CircleIcon = ({icon, message}: CircleIconProps) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  return (
+    <div
+      className="relative flex items-center justify-center"
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
```

Comment:
I think this will not work on mobile devices. The mockup recommends using a modal on small screens but I don't know if I like that idea. I think one alternative could be also listening to click/tap events.

Could we use [`@accessible/popover`](https://github.com/accessible-ui/popover) to handle the hover behavior instead?